File: 01.]ARRAY/javascript/001_Search_Element.js
function search(arr,element){
    for (let i = 0; i < arr.length; i++) {
       if ( element === arr[i]){
        return i
       }
     
    }
    return -1
}

const arr = [20, 5, 7, 25];
console.log(`Searched index= ${search(arr, 5)}`);

File: 01.]ARRAY/python/001_Search_Element.py
arr = [20, 5, 7, 25]
element = 5
def search(arr,element):
    for i,value in enumerate(arr):
        if element == value:
            return i
    return -1    
print("Array Searched:", arr)
print("Searching for:", element)
print("Searched index=", search(arr, element))

